packages/icons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/icons",
-  "version": "1.240.0",
+  "version": "1.240.1",
```

Comment:
this should be a minor version bump, since this is not a bug fix, but a new addition instead.

Reply:
updated